pages/am/3.x/admin-guide/admin-guide-organizations-and-environments.adoc

```diff
@@ -24,8 +24,8 @@ An environment belongs to one organization.
 == Organization
 An _organization_ allows users to customize AM to their own internal organization. It can manage multiple environments by sharing configurations like:
 
-* Users allows to access the AM Console
+* User permissions to access AM Console
```

Comment:
Not sure this modification is in the right PR :grin:

Reply:
This file is linked from the Cockpit doc, so I updated a couple of things while I was there.